src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -163,19 +172,36 @@ public static class PrereleaseCompatibilityTransformer
 
                     if (run["files"] is JObject files)
                     {
-                        keyToIndexMap = new Dictionary<string, int>();
+                        fileKeyToIndexMap = new Dictionary<string, int>();
 
                         run["files"] =
                             ConstructFilesArray(
                                 files,
-                                keyToIndexMap);
+                                fileKeyToIndexMap);
 
-                        modifiedLog |= keyToIndexMap.Count > 0;
+                        modifiedLog |= fileKeyToIndexMap.Count > 0;
                     }
 
+                    if (run["resources"] is JObject resources)
+                    {
+                        // Remove 'open' from rule configuration default level enumeration
+                        // https://github.com/oasis-tcs/sarif-spec/issues/288
+                        modifiedLog |= RemapRuleDefaultLevelFromOpenToNote(resources);
```

Comment:
>RemapRuleDefaultLevelFromOpenToNote [](start = 39, length = 35)

_Love_ the method name!

Reply:
Oh, it already existed.

---
In reply to: [249161656](https://github.com/Microsoft/sarif-sdk/pull/1223#discussion_r249161656) [](ancestors = 249161656)